common/test/acceptance/tests/lms/test_learner_profile.py

```diff
@@ -347,4 +356,149 @@ def test_user_is_under_age(self):
+        """
+        self.assert_default_image_has_public_access()
+
+        self.my_profile_page.upload_file(filename='image.jpg')
+        self.assertIsNone(self.my_profile_page.image_upload_success)
+        self.my_profile_page.visit()
+        self.assertIsNone(self.my_profile_page.image_upload_success)
+
+    def test_user_can_see_error_for_exceeding_max_file_size_limit(self):
+        """
+        Scenario: Upload profile image does not work for > 1MB image file.
+
+        Given that I am on my profile page with public access
+        And I can see default image
+        When I move my cursor to the image
+        Then i can see the upload/remove image text
+        When i upload new > 1MB image via file uploader
+        Then i can see the error message for file size limit
+        And i can still see the default image after page reload.
+        """
+        self.assert_default_image_has_public_access()
+
+        self.my_profile_page.upload_file(filename='larger_image.jpg')
+        self.assertEqual(self.my_profile_page.profile_image_message, "Your image must be smaller than 1 MB in size.")
+        self.my_profile_page.visit()
+        self.assertTrue(self.my_profile_page.profile_has_default_image)
+
+    def test_user_can_see_error_for_file_size_below_the_min_limit(self):
+        """
+        Scenario: Upload profile image does not work for < 100 Bytes image file.
+
+        Given that I am on my profile page with public access
+        And I can see default image
+        When I move my cursor to the image
+        Then i can see the upload/remove image text
+        When i upload new < 100 Bytes image via file uploader
+        Then i can see the error message for minimum file size limit
+        And i can still see the default image after page reload.
+        """
+        self.assert_default_image_has_public_access()
+
+        self.my_profile_page.upload_file(filename='list-icon-visited.png')
+        self.assertEqual(self.my_profile_page.profile_image_message, "Your image must be at least 100 Bytes in size.")
+        self.my_profile_page.visit()
+        self.assertTrue(self.my_profile_page.profile_has_default_image)
+
+    def test_user_can_see_error_for_wrong_file_type(self):
+        """
+        Scenario: Upload profile image does not work for wrong file types.
+
+        Given that I am on my profile page with public access
+        And I can see default image
+        When I move my cursor to the image
+        Then i can see the upload/remove image text
+        When i upload new csv file via file uploader
+        Then i can see the error message for wrong/unsupported file type
+        And i can still see the default image after page reload.
+        """
+        self.assert_default_image_has_public_access()
+
+        self.my_profile_page.upload_file(filename='cohort_users_only_username.csv')
+        self.assertEqual(self.my_profile_page.profile_image_message, "Unsupported file type.")
```

Comment:
Your image must be a .gif, .jpg, .png, or .tiff file.

(If any of those file types are incorrect, feel free to adjust the text as necessary - just make sure that the file types are written with a period and in lowercase letters.)